### ISSUE TITLE
Can't @import .sass Files

### ISSUE THREAD

**Issue by ivoilic** [babobski/Komodo-sass-compiler#2]
Komodo Edit, version 9.3.1, build 16424
Latest version of SASS Compiler

Cannot @import .sass files
"Error: File to import not found or unreadable"

Tried to compile bulma.sass without any modification from working build with no success.
 https://github.com/jgthms/bulma

Gave read, write, and execute permissions to both user and group but no success.

Please advise.

**Comment by babobski** [comment]
@ivoilic I looked in to this, there are 2 problems when compiling the bulma.sass.
The first one is the main bulma.sass file is using the wrong syntax on [imports](http://sass-lang.com/guide#topic-5), ti's using the scss syntax with qoutes, for sass this should be without qoutes. When i correct the syntax i still can't compile the file, beacause i noticed a bug in the compiler, currently it can't handle imports inside imports (the utility file has imports inside). 

Cheked the code of the sass compiler and found the bug, i forgot to pass the file extension to the included imports. Fixed this and now it's working as espected.
I will bring out a bug fix release later this day.

**Note** to compile bulma.sass you still have to correct the syntax inside the main file to the sass syntax to get it working.
